flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/operator/CEPOperatorTest.java

```diff
@@ -726,6 +727,50 @@ public void testCEPOperatorCleanupEventTimeWithSameElements() throws Exception {
 		}
 	}
 
+	@Test
+	public void testCEPOperatorSideOutputLateElementsEventTime() throws Exception {
+
+		Event startEvent = new Event(41, "c", 1.0);
+		Event middle1Event1 = new Event(41, "a", 2.0);
+		Event middle1Event2 = new Event(41, "a", 3.0);
+		Event middle1Event3 = new Event(41, "a", 4.0);
+
+		SelectCepOperator<Event, Integer, Map<String, List<Event>>> operator = CepOperatorTestUtilities.getKeyedCepOpearator(
+			false,
+			new ComplexNFAFactory());
+		OneInputStreamOperatorTestHarness<Event, Map<String, List<Event>>> harness = CepOperatorTestUtilities.getCepTestHarness(operator);
+
+		try {
```

Comment:
Could you switch it to `try-with-resources` block? I know there is a lot tests in that style, but we should add new tests with a proper style ;)